tools/solidity-coverage-example/contracts/TokenSend.sol

```diff
@@ -0,0 +1,44 @@
+// SPDX-License-Identifier: UNLICENSED
```

Comment:
Update all the sol files with `Apache-2.0`
```suggestion
// SPDX-License-Identifier: Apache-2.0
```

Reply:
Done